Build error with Clang on FreeBSD/powerpc64(le)

Building with GCC works fine, I just tried Clang (11.0.1) because OpenBLAS 0.3.11 supposedly fix it (I have 0.3.13 now).

Exact error is:
```
cc -O2 -pipe  -fstack-protector-strong -fno-strict-aliasing  -O2 -DMAX_STACK_ALLOC=2048 -DHAVE_P10_SUPPORT -Wall -m64 -DF_INTERFACE_GFORT -fPIC -DDYNAMIC_ARCH -DDYNAMIC_OLDER -DNO_AVX -DNO_AVX2 -DSMP_SERVER -DNO_WARMUP -DMAX_CPU_NUMBER=64 -DMAX_PARALLEL_NUMBER=1 -DBUILD_SINGLE=1 -DBUILD_DOUBLE=1 -DBUILD_COMPLEX=1 -DBUILD_COMPLEX16=1 -DVERSION=\"0.3.13\" -Ofast -mvsx -fno-fast-math -mcpu=power9 -mtune=power9 -DUSE_OPENMP -fopenmp -UASMNAME -UASMFNAME -UNAME -UCNAME -UCHAR_NAME -UCHAR_CNAME -DASMNAME= -DASMFNAME=_ -DNAME=_ -DCNAME= -DCHAR_NAME=\"_\" -DCHAR_CNAME=\"\" -DNO_AFFINITY -I. -O2 -DMAX_STACK_ALLOC=2048 -fopenmp -DHAVE_P10_SUPPORT -Wall -m64 -DF_INTERFACE_GFORT -fPIC -DDYNAMIC_ARCH -DDYNAMIC_OLDER -DNO_AVX -DNO_AVX2 -DSMP_SERVER -DUSE_OPENMP -DNO_WARMUP -DMAX_CPU_NUMBER=64 -DMAX_PARALLEL_NUMBER=1 -DBUILD_SINGLE=1 -DBUILD_DOUBLE=1 -DBUILD_COMPLEX=1 -DBUILD_COMPLEX16=1 -DVERSION=\"0.3.13\" -Ofast -mvsx -fno-fast-math -mcpu=power9 -mtune=power9 -DUSE_OPENMP -fopenmp -UASMNAME -UASMFNAME -UNAME -UCNAME -UCHAR_NAME -UCHAR_CNAME -DASMNAME=dynamic_power -DASMFNAME=dynamic_power_ -DNAME=dynamic_power_ -DCNAME=dynamic_power -DCHAR_NAME=\"dynamic_power_\" -DCHAR_CNAME=\"dynamic_power\" -DNO_AFFINITY -I../..   -c -o dynamic_power.o dynamic_power.c
dynamic_power.c:43:6: error: use of unknown builtin '__builtin_cpu_is' [-Wimplicit-function-declaration]
        if (__builtin_cpu_is("power6") || __builtin_cpu_is("power6x"))
            ^
dynamic_power.c:52:6: error: use of unknown builtin '__builtin_cpu_supports' [-Wimplicit-function-declaration]
        if (__builtin_cpu_supports ("arch_3_1") && __builtin_cpu_supports ("mma"))
            ^
dynamic_power.c:52:6: note: did you mean '__builtin_cpu_is'?
dynamic_power.c:43:6: note: '__builtin_cpu_is' declared here
        if (__builtin_cpu_is("power6") || __builtin_cpu_is("power6x"))
            ^
2 errors generated.
```

--- a/driver/others/dynamic_power.c
+++ b/driver/others/dynamic_power.c
@@ -40,7 +40,14 @@ char *gotoblas_corename(void) {
 	return corename[0];
 }
 
-#ifdef C_PGI
+#if defined(__clang__)
+static int __builtin_cpu_supports(char* arg) 
+{
+	return 0;
+}
+#endif
+
+#if defined(C_PGI) || defined(__clang__)
 /*
  * NV HPC compilers do not yet implement __builtin_cpu_is().
  * Fake a version here for use in the CPU detection code below.

